docs/blog/2022-06-24-junopi/index.mdx

```diff
@@ -0,0 +1,127 @@
+---
+# slug: junopi
+# title: Running Juno from your Raspberry Pi
+title : Blog template 2 (WIP)
+# authors: [depablos, maceo]
+# tags: [juno, rpi, raspberry, pi, deploy]
+---
+
+<!-- Juno is a node which aims to help **decentralise StarkNet, a prominent Ethereum Layer 2**.
+
```

Comment:
All of this will not appear right? Will it just say WIP?

Reply:
yeah